github_connector_odoo/models/odoo_module_version.py

```diff
@@ -0,0 +1,286 @@
+
+    author_ids = fields.Many2many(
+        string='Authors', comodel_name='odoo.author',
+        relation='github_module_version_author_rel',
+        column1='module_version_id', column2='author_id',
+        compute='_compute_author_ids')
+    # , store=True : Commented to optimize analysis.
+
+    lib_python_ids = fields.Many2many(
+        comodel_name='odoo.lib.python', string='Python Lib Dependencies',
+        relation='module_version_lib_python_rel', column1='module_version_id',
+        column2='lib_python_id', multi='lib', compute='_compute_lib',
+        store=True)
+
+    lib_bin_ids = fields.Many2many(
+        comodel_name='odoo.lib.bin', string='Bin Lib Dependencies',
+        relation='module_version_lib_bin_rel', column1='module_version_id',
+        column2='lib_bin_id', multi='lib', compute='_compute_lib', store=True)
+
+    odoo_type = fields.Selection(
+        string='Odoo Type', selection=_ODOO_TYPE_SELECTION, store=True,
+        compute='_compute_odoo_type')
+
+    # Overload Section
+    @api.multi
+    def unlink(self):
+        # Analyzed repository branches should be reanalyzed
+        if not self._context.get('dont_change_repository_branch_state', False):
+            repository_branch_obj = self.env['github.repository.branch']
+            repository_branch_obj.search([
+                ('id', 'in', self.mapped('repository_branch_id').ids),
+                ('state', '=', 'analyzed')]).write({'state': 'to_analyze'})
+        return super(OdooModuleVersion, self).unlink()
+
+    # Compute Section
+    @api.multi
+    @api.depends('repository_branch_id.repository_id.name')
+    def _compute_odoo_type(self):
+        for version in self:
+            repository_name = version.repository_branch_id.repository_id.name
+            if repository_name.startswith('l10n-'):
+                version.odoo_type = 'localization'
+            elif repository_name.startswith('connector-'):
+                version.odoo_type = 'connector'
+            elif repository_name.startswith('vertical-'):
+                version.odoo_type = 'verticalization'
+            else:
+                version.odoo_type = 'other'
+
+    @api.multi
+    @api.depends('technical_name', 'repository_branch_id.complete_name')
+    def _compute_complete_name(self):
+        for version in self:
+            version.complete_name =\
+                version.repository_branch_id.complete_name +\
+                '/' + version.technical_name
+
+    @api.multi
+    @api.depends('description_rst')
+    def _compute_description_rst_html(self):
+        for version in self:
+            if version.description_rst:
+                try:
+                    output = publish_string(
+                        source=version.description_rst,
+                        settings_overrides=self._SETTING_OVERRIDES,
+                        writer=MyWriter())
+                except:
+                    output =\
```

Comment:
strangely with our current template (no description key in manifest), this will always return false with OCA modules